Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'fast_gettext', '~> 0.8'
 gem 'gettext_i18n_rails', '~> 1.0'
 gem 'i18n', '~> 0.6.4'
 gem 'turbolinks', '~> 2.5'
+gem 'doorkeeper'
```

Comment:
my experience with doorkeeper is that they change things quite a bit (maybe it changed as the library matured), anyway, I recommend requiring a certain major and minor version (leave patch levels up to bundler)
this way we won't get surprised by everything breaking when doorkeeper updates.